types/index.d.ts

```diff
@@ -56,24 +51,45 @@ type Content =
   | 'space-around'
   | 'space-evenly';
 
-export interface LayoutProps {
-  /** default to 'row' */
-  direction?: Direction;
-  pad?: boolean | number;
-  wrap?: boolean;
+export interface BlockProps {
   grow?: boolean;
   inline?: boolean;
   flex?: boolean | number;
   /** default to 'stretch' */
-  align?: Align;
   alignSelf?: Align;
+}
+
+export interface FlexProps extends BlockProps {
+  /** default to 'row' */
+  direction?: 'row' | 'column';
```

Comment:
I'd make it a type

Reply:
i was just matching https://github.com/4Catalyzer/layout/blob/master/src/PropTypes.js#L30

Reply:
up to you